tensorflow/contrib/data/python/ops/interleave_ops.py

```diff
@@ -240,9 +240,9 @@ def select_dataset(logits, seed):
   selector_input = dataset_ops.Dataset.zip(
       (logits_ds, random_ops.RandomDataset(seed).batch(2))).map(select_dataset)
 
-  logging.warn('selector_input.output_types: %s', selector_input.output_types)
-  logging.warn('selector_input.output_shapes: %s', selector_input.output_shapes)
+  logging.warn('selector_input.output_types: %s', str(selector_input.output_types))
+  logging.warn('selector_input.output_shapes: %s', str(selector_input.output_shapes))
   for i, dataset in enumerate(datasets):
-    logging.warn('dataset %i output_types: %s' % (i, dataset.output_types))
-    logging.warn('dataset %i output_shapes: %s' % (i, dataset.output_shapes))
+    logging.warn('dataset %i output_types: %s' % (i, str(dataset.output_types)))
+    logging.warn('dataset %i output_shapes: %s' % (i, str(dataset.output_shapes)))
```

Comment:
(Not sure if you're still debugging this, but...) Can you please remove these `logging.warn()` calls before submitting?

Reply:
I am still debugging...I've just verified that the types and shapes of the datasets to `DirectedInterleaveDataset` are now the same between Windows and Linux, but I'm still getting the `kernel not registered` error. Any other suggestions for what might be causing this?